src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIResponse.java

```diff
@@ -112,22 +128,32 @@ private static void validateRestApiResponse(String responseBody) {
     }
```

Comment:
We should remove the RetryableException and NonRetyableException classes completely and use the ServiceNowApiException object directly as used below.